wesnoth-did.po

```diff
@@ -6344,45 +6365,55 @@ msgid ""
 "This attack loses one strike on offense but deals three times the normal "
 "damage."
 msgstr ""
+"Počet útoků se při ofenzivním použití snižuje o jeden, ale každý útok "
+"způsobuje trojnásobné zranění."
 
 #. [attacks]: id=did_corpse_launcher
 #: data/campaigns/Descent_Into_Darkness/utils/abilities.cfg:541
 msgid "corpse launcher"
-msgstr ""
+msgstr "vymršťovač mrtvol"
 
+# Asi bych ten útok potřeboval vidět v praxi.
 #. [attacks]: id=did_corpse_launcher
 #: data/campaigns/Descent_Into_Darkness/utils/abilities.cfg:542
+#, fuzzy
 msgid ""
 "On offense, enemies cannot retaliate against this attack. This attack "
 "consumes 3 HP from each nearby allied corpse to deal 11 impact damage (per "
 "corpse) to the target on top of this attack’s base damage."
 msgstr ""
+"Při ofenzivním použití se nemohou nepřátelé proti tomuto útoku bránit "
+"protiútokem. Z každé sousedící spřátelené chodící mrtvoly se vytáhnou 3 "
+"životy, které se zformují do útoku úderem v síle 11 (za každou mrtvolu) do "
+"cíle na horní hranici základního poškození tohoto útoku."
 
 #. [leadership]: id=did_consumption
 #: data/campaigns/Descent_Into_Darkness/utils/abilities.cfg:668
 msgid "consumption"
-msgstr ""
+msgstr "sžírání"
 
 #. [leadership]: id=did_consumption
 #: data/campaigns/Descent_Into_Darkness/utils/abilities.cfg:669
 msgid ""
 "At night, this unit and adjacent allied undead gain 25% damage. However, "
 "this unit loses 2 HP per turn."
 msgstr ""
+"V noci způsobuje tato jednotka a všechni sousedící spřátelení nemrtví o 25 % "
+"větší zranění. Nicméně, tato jednotka ztrácí každé kolo 2 životy."
 
 #. [heals]: id=did_corpse_master
 #: data/campaigns/Descent_Into_Darkness/utils/abilities.cfg:730
-#, fuzzy
-#| msgid "Yes, master."
 msgid "corpse master"
-msgstr "Ano, pane."
+msgstr "mrtvolné mistrovství"
```

Comment:
```suggestion
msgstr "pán mrtvol"
```

Reply:
Nebude název schopnosti Pán mrtvol znít divně a špatně se pasovat do ostatních řetězců, kde je corpse master?

Reply:
Ty řetězce jsou jen dva a jen v DiD ("Gains the corpse master ability" a "corpse master"). Nechám to na vás.